packages/lit-html/src/lib/lit-html.ts

```diff
@@ -92,6 +97,10 @@ type ResultType = typeof HTML_RESULT | typeof SVG_RESULT;
  */
 export type TemplateResult = {
   _$litType$: ResultType;
+  // TODO (justinfagnani): consider shorter names, like `s` and `v`. This is a
+  // semi-public API though. We can't just let Terser rename them for us,
+  // because we need TemplateResults to work between compatible versions of
+  // lit-html.
```

Comment:
Hmm, how about an array?

Arg, except it needs the branding...
`['$litResult$', type, strings, values]`?

Reply:
Yeah, I think object will be better overall, in part because of the branding.